src/sema/ast.rs

```diff
@@ -14,7 +14,7 @@ use num_rational::BigRational;
 use once_cell::unsync::OnceCell;
 pub use solang_parser::diagnostics::*;
 use solang_parser::pt;
-use solang_parser::pt::{CodeLocation, FunctionTy, OptionalCodeLocation};
+use solang_parser::pt::{CodeLocation, FunctionTy, Identifier, OptionalCodeLocation};
```

Comment:
The problem with this is, now we have both `pt::Identifier` in this file and `Identifier`. Please make the file consistently use one or the other.